lib/InputOutputItemsUtility.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:io';
+import 'Item.dart';
+
+enum ItemTypes { raw, manufactured, imported }
+
+class InputOutputItemsUtility {
```

Comment:
Utility class shouldn't hold member variables
Create an execution or application class which is called from main()
Main() is a starting point 
Direct it to run the application 
The application should be the class above all 
Take input, Validate input, Process input, output the result (Delegate these)